pkg/controller/util.go

```diff
@@ -762,6 +764,11 @@ func getPriorityClass(pvc *v1.PersistentVolumeClaim) string {
 	return anno[AnnPriorityClassName]
 }
 
+func isPodRetainAfterCompletion(pvc *corev1.PersistentVolumeClaim) bool {
+	retain, exists := pvc.ObjectMeta.Annotations[AnnPodRetainAfterCompletion]
```

Comment:
Are we 100% sure pvc.ObjectMeta.Annotations is not nil?

Reply:
Afaik, nowhere in our code we check Annotations for nil, so I assumed it's sure initialized, but I'll check that if needed.
Anyway a nicer line will sure be using `pvc.GetAnnotations()` and not `ObjectMeta`.

Reply:
it is okay to get values from a nil map.  Just have to check nil when setting